pkg/mount/block/blockdev_test.go

```diff
@@ -6,33 +6,259 @@ package block
 
 import (
 	"fmt"
+	"path/filepath"
 	"reflect"
+	"strings"
 	"testing"
 
 	"github.com/u-root/u-root/pkg/pci"
 	"github.com/u-root/u-root/pkg/testutil"
 )
 
-// TestFindMountPointNotExists checks that non existent
-// entry is checked and nil returned
-func TestFindMountPointNotExists(t *testing.T) {
-	LinuxMountsPath = "testdata/mounts"
-	if _, err := GetMountpointByDevice("/dev/mapper/sys-oldxxxxxx"); err == nil {
-		t.Errorf(`GetMountpointByDevice("/dev/mapper/sys-oldxxxxxx") = _, %v, want not nil`, err)
+func TestDebug(t *testing.T) {
+	Debug("foo")
+}
```

Comment:
I cant point my finger on it, but somehow I'm unhappy with this. Does this add any value?

Reply:
Same here, it just covers
``` golang
// Debug function to override for verbose logging.
Debug = func(string, ...interface{}) {}
```
in blockdev_linux.go